reference/security-policy.md

```diff
@@ -0,0 +1,25 @@
+(ref-security-policy)=
+# Security policy
+
+Learn about our [release and support policy](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/release-notes/release-notes/#release-and-support-policy) for the nature of our releases and versions.
```

Comment:
Should we add in the linked document what we have defined [here](https://docs.google.com/document/d/1IkAanYou9y5sUIaLZC8JrIVTmpcBzi3JyE_bWT73dUo/edit?tab=t.0#heading=h.jf2awfdyd6r1) to let users know what we're going to fix and what not?

Reply:
I have now included details about scoring and our response process to the same document as this is the right place to talk about that.

See [line #24](https://github.com/canonical/anbox-cloud-docs/pull/232/files#diff-ae73e90592369962b66aa1444c3783ce5f560913042994c4d49ed1ad948c82a5R24)